From 02c810a47ca6c3aadb0226ca4cad07f48490e12e Mon Sep 17 00:00:00 2001
From: shucheng <shucheng@bd-apaas.com>
Date: Wed, 22 Mar 2023 20:34:55 +0800
Subject: [PATCH] yaml

---
 go.mod | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/go.mod b/go.mod
index 013c38e6..ea88cff2 100644
--- a/go.mod
+++ b/go.mod
@@ -2,6 +2,10 @@ module github.com/drone/drone
 
 replace github.com/docker/docker => github.com/docker/engine v17.12.0-ce-rc1.0.20200309214505-aa6a9891b09c+incompatible
 
+replace github.com/drone/drone-yaml => github.com/open-beagle/drone-yaml v1.2.4-beagle.0
+
+replace github.com/drone/drone-ui => github.com/open-beagle/drone-ui v2.9.1-beagle.0+incompatible
+
 require (
 	github.com/99designs/httpsignatures-go v0.0.0-20170731043157-88528bf4ca7e
 	github.com/Azure/azure-storage-blob-go v0.7.0
-- 
2.25.1

