From f6d90c2b113ad24e301458280a112ea1e8e06dc0 Mon Sep 17 00:00:00 2001
From: shucheng <shucheng@bd-apaas.com>
Date: Mon, 27 Mar 2023 15:39:44 +0800
Subject: [PATCH] license

---
 service/license/load.go | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/service/license/load.go b/service/license/load.go
index 8163dd31..fff0fe6e 100644
--- a/service/license/load.go
+++ b/service/license/load.go
@@ -12,8 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// +build !nolimit
-// +build !oss
+//go:build !nolimit && !oss
+// +build !nolimit,!oss
 
 package license
 
@@ -27,7 +27,7 @@ import (
 )
 
 // embedded public key used to verify license signatures.
-var publicKey = []byte("GB/hFnXEg63vDZ2W6mKFhLxZTuxMrlN/C/0iVZ2LfPQ=")
+var publicKey = []byte("ajmJ+hb2Nvx+1yKIf0kWjROtWD0hjG8ChE5JljtX0us=")
 
 // License renewal endpoint.
 const licenseEndpoint = "https://license.drone.io/api/v1/license/renew"
@@ -83,6 +83,7 @@ func Load(path string) (*core.License, error) {
 		dat := new(core.License)
 		json.Unmarshal(decoded.Dat, dat)
 		license.Repos = dat.Repos
+		license.Kind = dat.Kind
 	}
 
 	return license, err
-- 
2.35.1

